src/packing_queue/tables/PackingQueueTable.jsx

```diff
@@ -459,8 +464,16 @@ const PackingQueueTable = ({
           <tr>
             <TablePagination
               count={queueData.length}
-              rowsPerPageOptions={[numRowsPerPage]}
+              rowsPerPageOptions={PAGINATION_SIZING_OPTIONS}
               rowsPerPage={numRowsPerPage}
+              onRowsPerPageChange={(event) => {
```

Comment:
This logic is repeated alot, should we make a little helper fn?

Reply:
Reduced redundancy